### PR TITLE
feat: Implement dual promo component

### PR DESCRIPTION
**Pull request by ChesneyJulian**
image
image

### CODE REVIEW

File: packages/ember-test-app/app/templates/promo.hbs [hunk 1]
```diff
@@ -1,3 +1,115 @@
-<div class="container p-3">
+{{! <div class="container p-3">
   <Mktg::Promo />
-</div>
\ No newline at end of file
+</div> }}
```
Comment (TSenter) [inline packages/ember-test-app/app/templates/promo.hbs:3]: You can delete this

File: packages/ember-test-app/app/templates/promo.hbs [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+<main data-theme="marketing" class="bg-primary-subtle vw-100">
```
Comment (TSenter) [inline packages/ember-test-app/app/templates/promo.hbs:4]: Not 100% sure why, but I get a horizontal scroll bar when running locally. It looks like this is too wide?

**Review by TSenter — CHANGES_REQUESTED** [review]
Is this component have to be limited to two items? Could it be modified to yield an internal `Promo` component, and have an option to specify how many are rendered in one row? If we have a `Promo` and `DualPromo` component, I feel like it's only a matter of time before we need a `TriplePromo` component.

**Comment by ChesneyJulian** [comment]
> Is this component have to be limited to two items? Could it be modified to yield an internal `Promo` component, and have an option to specify how many are rendered in one row? If we have a `Promo` and `DualPromo` component, I feel like it's only a matter of time before we need a `TriplePromo` component.

That makes sense, I was probably too focused on the figma designs. So a better route could be making the base promo component have an optional orientation parameter for vertical or horizontal, then this component would be a promo container that has options for organizing content.

File: packages/ember-test-app/app/templates/promo.hbs (hunk 2 above)
Comment (ChesneyJulian) [reply packages/ember-test-app/app/templates/promo.hbs:4]: Yeah I can remove the vw-100, I was doing that for local developement but forgot to remove it!

**Pull request closed without merging**